### PR TITLE
rook:PR for Fix for bug 57954

### PR DESCRIPTION
**Pull request by gaord**
NFSRados constructor expects type of rados as a parameter instead of MgrModule. This change should fix nfs cluster creation bug 57954.

Bug to fix: https://tracker.ceph.com/issues/57954
Signed-off-by: Ben Gao <bengao168@msn.com>

### CODE REVIEW

**Comment by travisn** [comment]
@BlaineEXE ptal

**Comment by adk3798** [comment]
> Change looks OK to me, but the commit message is a little out of the norm, but I haven't worked on ceph long enough to know how much the ceph team is a stickler for that kind of thing.... any thoughts @adk3798 ?

typically we'd want the commit message to start with "mgr/rook: " here to make it clear what the commit is touching (sort of similar to the title on this PR). It won't entirely block merging but it would be nice to have if possible @gaord . Other thant could nitpick about splitting it up into a proper commit title and message instead of one long line but I don't care too much personally.

**Comment by BlaineEXE** [comment]
As long as this is working, it seems good to me

**Comment by gaord** [comment]
> > Change looks OK to me, but the commit message is a little out of the norm, but I haven't worked on ceph long enough to know how much the ceph team is a stickler for that kind of thing.... any thoughts @adk3798 ?
> 
> typically we'd want the commit message to start with "mgr/rook: " here to make it clear what the commit is touching (sort of similar to the title on this PR). It won't entirely block merging but it would be nice to have if possible @gaord . Other thant could nitpick about splitting it up into a proper commit title and message instead of one long line but I don't care too much personally.

just get the commit message get updated as designated. Thanks.

**Comment by gaord** [comment]
@travisn anything else for this bit of work to proceed with the pipeline? Hopefully it could be included in the next point release of Quincy.

**Review by travisn — APPROVED** [review]

**Comment by opencmit2** [comment]
Hi, Just wonder if there is anything needs to be done for the work to go into the Quincy point release after approval. @travisn

**Comment by opencmit2** [comment]
Hi, Wonder anybody could merge this to Quincy point release? Thanks. @travisn

**Comment by travisn** [comment]
@jmolmo Want to review and backport to quincy? thanks!

**Pull request merged**
